.github/workflows/release.yml

```
@@ -302,15 +302,23 @@ jobs:
     name: Update on Homebrew-Core
     needs: [sleep-before-homebrew] # Needs to run after pypi released so brew can update pypi dependency hashes
     runs-on: macos-10.15
-    strategy:
-      matrix:
-        python: ["3.9.12"]
     steps:
       - name: Setup Python
         uses: actions/setup-python@v1
         with:
-          # Needs to be < 3.10 see https://github.com/returntocorp/semgrep/issues/4213
-          python-version: ${{ matrix.python }}
+          # Pin to python 3.9 for pipgrip issues: see https://github.com/returntocorp/semgrep/issues/4213
+          python-version: '3.9.*'
```

```suggestion
          python-version: '3.9.x'
```

I'm sure that worked as well, but this format matches the documentation at https://github.com/actions/setup-python

---

change applied